src/main/java/model/FruitTransaction.java

```diff
@@ -0,0 +1,55 @@
+package model;
+
+public class FruitTransaction {
+    private Operation operation;
+    private String fruit;
+    private int quantity;
+
+    public Operation getOperation() {
+        return operation;
+    }
+
+    public void setOperation(String op) {
+        this.operation = switch (op) {
+            case "b" -> Operation.BALANCE;
+            case "s" -> Operation.SUPPLY;
+            case "p" -> Operation.PURCHASE;
+            case "r" -> Operation.RETURN;
+            default -> throw new IllegalArgumentException(
+                    "There is no Enum element, corresponding to the argument " + op);
+        };
+    }
+
+    public String getFruit() {
+        return fruit;
+    }
+
+    public void setFruit(String fruit) {
+        this.fruit = fruit;
+    }
+
+    public int getQuantity() {
+        return quantity;
+    }
+
+    public void setQuantity(int quantity) {
+        this.quantity = quantity;
+    }
+
+    public enum Operation {
+        BALANCE("b"),
+        SUPPLY("s"),
+        PURCHASE("p"),
+        RETURN("r");
+
+        private final String transaction;
+
+        Operation(String lineFromFile) {
+            transaction = lineFromFile;
+        }
+
+        public String getTransaction() {
```

Comment:
The method `getTransaction()` is not a good name. It's not clear what it does. It would be better to name it `getOperationSymbol()` or similar.